wiki/Tournaments/CWC/1/en.md

```diff
@@ -268,9 +268,7 @@ Monday, 5 December 2011:
 | Group | Team 1 |  |  | Team 2 | Match link |
 | --: | --: | :-: | :-: | :-- | :-- |
 | B | **State(s) of the Art** | **2** | 1 | Hungary CTB | [#1](https://osu.ppy.sh/community/matches/2123665) |
-| D | Chile | *N/A* | *N/A* | China | *not played¹* |
-
-¹ *As per the Tournament Management's decision ([\[1\]](https://osu.ppy.sh/community/forums/posts/1184327) [\[2\]](https://osu.ppy.sh/community/forums/posts/1206142) [\[3\]](https://osu.ppy.sh/community/forums/posts/1187916)), the Chile vs. China match was not played.*
+| D | Chile | *N/A* | *N/A* | China | *not played[^cl-cn-skip]* |
```

Comment:
i don't think the footnote superscript should have any formatting. currently there are a lot of instances of superscripts in italics like this

with: normal[^id] *italic[^id]* **bold[^id]** ***italic and bold[^id]*** ~~strikethrough[^id]~~ [link[^link]](https://example.com)
 
without: normal[^id] *italic*[^id] **bold**[^id] ***italic and bold***[^id] ~~strikethrough~~[^id] [link](https://example.com)[^id]

[^id]: thing
[^link]: it's separately clickable although it doesn't look like it 🤔